### PR TITLE
applier.groovy - update to take playbookAdditionalArgs and clusterAPI and clusterToken

### PR DESCRIPTION
**Pull request by itewk**

### CODE REVIEW

**Comment by itewk** [comment]
@etsauer could i get a review?

**Comment by itewk** [comment]
@pabrahamsson could I get a review?

File: vars/applier.groovy [hunk 1]
```diff
@@ -1,36 +1,52 @@
...
-    String clusterURL = ""
-    String inventoryPath = ""
-    String requirementsPath = ""
-    String skipTlsVerify = false
-    String rolesPath = "galaxy"
-    String applierPlaybook = "galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml"
+    String inventoryPath          = ''
+    String requirementsPath       = ''
+    String ansibleRootDir         = ''
+    String rolesPath              = 'roles'
```
Comment (etsauer) [inline vars/applier.groovy]: please change to 'galaxy'. that's been the applier naming standard since its not simply a role.

Comment (itewk) [reply vars/applier.groovy]: i'll change it back, but it didn't work for us with client when using it with galaxy, ansible couldn't find the role, only thing that worked was changing that value to roles.

Comment (itewk) [reply vars/applier.groovy]: @etsauer done.

**Comment by itewk** [comment]
@etsauer did the update.

**Pull request merged**